schemas/snuba-uptime-results.v1.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "snuba_uptime_result",
+  "definitions": {
```

Comment:
these are copied from https://github.com/getsentry/sentry-kafka-schemas/blob/6702856230dfaf3b15210cf2894cbb5106950ca3/schemas/uptime-results.v1.schema.json#L5

i tried to link directly to them, but it doesn't seem like our framework supports that.

Reply:
@lynnagara do you know if it's possible to cross reference schemas somehow in this repo?

Would be nice to not need to duplicate these

Reply:
@wedamija is dealing with this in #360

Reply:
Oh cool, I totally mised it. Thanks!